File: cfgs/java_cfgs_foo/COUNT_TRAILING_ZEROES_FACTORIAL_NUMBER.java
Nodes
1;special;;;
2;statement;int count = 0;14;
3;statement;int i = 5;15;
4;conditional;(foo_1 / foo_2) >= 1;16;
5;statement;count += foo_1 / foo_2;17;
6;statement;i *= 5;17;
7;statement;return foo_1;18;
8;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;7;
4;6;
5;4;
6;8;
7;

File: cfgs/pythoncfgs/COUNT_TRAILING_ZEROES_FACTORIAL_NUMBER.py
Nodes
1;statement;foo_1 = 0;8;
2;statement;foo_1 = 5;9;
3;conditional;foo_1 / foo_2 >= 1;10;
4;statement;return fun_1(foo_1);13;
5;statement;foo_1 += fun_1(foo_2 / foo_3);11;
6;statement;foo_1 *= 5;12;
7;exit;;
Edges
0;2;
1;3;
2;4;5;
3;7;
4;6;
5;3;
6;